ocs_ci/ocs/version.py

```diff
@@ -275,3 +276,20 @@ def main():
         return
 
     report_ocs_version(cluster_version, image_dict, file_obj=sys.stdout)
+
+
+def if_version(expressions):
```

Comment:
Could you please add some doc string with explanation here?

Reply:
done, thanks!